pyproject.toml

```diff
@@ -147,7 +147,8 @@ testpaths = ["tests/"]
 markers = ["django", "starlette", "channels"]
 asyncio_mode = "auto"
 filterwarnings = [
-    "ignore::DeprecationWarning:strawberry.*.resolver"
+    "ignore::DeprecationWarning:strawberry.*.resolver",
+    "ignore::DeprecationWarning:LazyType is deprecatedy",
```

Comment:
Shouldn't it be `deprecated`?

Reply:
yes :D also I'm not sure it is actually working :D